test/claim.test.ts

```diff
@@ -9,12 +9,13 @@ describe('Tests for claim operation', () => {
 
     beforeEach(async () => {
         sdk = new MockRelayingServices();
+        sdk.initialize({});
     });
 
     it('Should fail by not implemented error', async () => {
         try {
             await sdk.claim({});
```

Comment:
As pointed out here https://github.com/infuy/relaying-services-sdk/pull/2/files#r701720209, I would add a `fail` statement to avoid false positive